### PR TITLE
DATAUP-313 Fix tests and PR comments

### PR DESCRIPTION
**Pull request by eamahanna**
# Description of PR purpose/changes
This PR comments out the two file path widget button tests. These tests have been flaky despite the functionality working in the UI. I want to get the tests running in ci again, so this is a band-aid until I can take a closer look at what is happening. 

I also add margin to a cell in the job status tab because I noticed it was smushed when the screen was small. 

# Jira Ticket / Issue #
e.g. https://kbase-jira.atlassian.net/browse/DATAUP-X
- [ ] Added the Jira Ticket to the title of the PR e.g. (DATAUP-69 Adds a PR template)

# Testing Instructions
* Details for how to test the PR: 
- [ ] Tests pass in travis and locally 
- [ ] Changes available by spinning up a local narrative and navigating to _X_ to see _Y_

# Dev Checklist:

- [ ] My code follows the guidelines at https://sites.google.com/truss.works/kbasetruss/development
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have run Black and Flake8 on changed Python Code manually or with git precommit (and the travis build passes)

# Updating Version and Release Notes (if applicable)

- [ ] [Version has been bumped](https://semver.org/) for each release
- [ ] [Release notes](/RELEASE_NOTES.md) have been updated for each release (and during the merge of feature branches)

### CODE REVIEW

File: kbase-extension/scss/partials/_bulkImportCell.scss [hunk 1]
```diff
@@ -268,6 +268,7 @@ $button-outdated: rgb(247, 155, 34);
 
   &__button--add_row {
     margin-left: 10px;
+    margin-bottom: 5px;
```
Comment (eamahanna) [inline kbase-extension/scss/partials/_bulkImportCell.scss:271]: add space under button: request from Mallory

File: kbase-extension/scss/partials/_job-status.scss [hunk 2]
```diff
@@ -24,7 +24,7 @@
       background: use-color('primary-lighter');
 
       &:hover {
-        background: use-color('primary-lighter');
+        background: rgb(242,239,235);
```
Comment (eamahanna) [inline kbase-extension/scss/partials/_job-status.scss:27]: Used the incorrect color

File: test/unit/spec/common/cellComponents/filePathWidget-Spec.js [hunk 3]
```diff
@@ -105,10 +105,10 @@ define([
                     parameters: parameters,
                 })
                 .then(() => {
-                    let preClickNumberOfRows = $('tr').length;
+                    let preClickNumberOfRows = $(node).find('tr').length;
```
Comment (eamahanna) [inline test/unit/spec/common/cellComponents/filePathWidget-Spec.js:108]: Something was hanging in the dom after each test. I needed to specify the node so Karma didn't get confused when running all tests.

File: kbase-extension/scss/partials/_job-status.scss (hunk 2 above)
Comment (ialarmedalien) [reply kbase-extension/scss/partials/_job-status.scss:27]: That colour is `base-lightest`

**Review by ialarmedalien — APPROVED** [review]
Ship it!

**Pull request merged**